package.json

```diff
@@ -26,11 +26,11 @@
     "csvjson-csv2json": "^5.0.6",
     "electron-store": "^8.0.1",
     "frappe-charts": "1.6.1",
-    "frappejs": "https://github.com/frappe/frappejs",
+    "frappejs": "frappe/frappejs",
     "knex": "^0.95.12",
     "lodash": "^4.17.21",
     "luxon": "^2.0.2",
-    "sqlite3": "^5.0.2",
+    "sqlite3": "npm:@vscode/sqlite3@^5.0.7",
```

Comment:
Why are these chagnes showing up in the PR @piyushsinghania ?

Reply:
I had taken a pull from frappe master and since the base branch of this PR is different it is showing like that. 

You can take a pull from frappe master and then I will change the base branch.